Scripts/01 Get Location Data.R

```diff
@@ -0,0 +1,352 @@
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Services mapping script 1 - Primary and Secondary Care locations
+
+# This script extracts data on primary and secondary care locations from 
+# publicly available resources and aggregates them into a consistent format
+# for mapping. 
+
+# Specifically:
+# - GP practices
+# - Care homes
+# - Hospitals (All types)
+# - Pharmacies
+
+# The main data source is opendata.nhs.scot (operated by PHS), however, it also
+# uses the Scottish Care inspectorate (careinspectorate.com) and National
+# Records of Scotland data to link service data together. The script has been
+# structured to largely function automatically however, some URLs may need to
+# be updated as geographic boundaries/relationships change over time.
+
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+
+# 1 Housekeeping, setup, etc. ---------------------------------------------
+
+library(tidyverse) # General data maniupulation package
+library(tidylog) # Accompanies tidyverse
+library(phsopendata) # OpenData API
+library(ckanr) # Web scraping 
+library(rvest) # Web scraping 
+
+gc() # Tidying environment
+
+
+# 2. Loading location data ------------------------------------------------
+
+## 2.1. GP Data ----
+
+# Identify most recent data
+most_recent_practice_sizes = package_show(id = "gp-practice-contact-details-and-list-sizes", url = "https://www.opendata.nhs.scot/", as = "table")[["resources"]] %>%
```

Comment:
Instead of this you could simply use the get_latest_resource() function from phsopendata